### DISCUSSION TITLE
Updated

### DISCUSSION THREAD

**Discussion by Dumdididum** [apohl79/audiogridder#1292] [Help]
Since I update AG to 1.2.0, it seems some of my plugins are not being recognized anymore. Am I doing something wrong?

**Comment by emack999** [comment]
Have you tried wiping your plugin cache and rescanning?

**Comment by Dumdididum** [comment]
Yeah, I've tried clearing it and rescanning, but still don't get a lot of
the plugins. I used to be able to load a plugin called Chainer, which is a
plugin that loads a "chain" of other plugins. That way I could load other
plugins into it. But now it doesn't seem to be recognized by AG anymore.


*William Fleming*
*FlemingSound*
*(989)400-0140 <http://voice.google.com/calls?a=nc,%2B19894000140>*
***@***.*** ***@***.***>*
